src/routes/StackView/StackView.tsx

```diff
@@ -41,6 +41,19 @@ export const StackView = () => {
     return null;
   }
 
```

Comment:
When Parent note gets deleted the stackview gets rerendered and has no noteId in the params anymore. 
So the stack view navigates back to board (line 39 - 41).